include/sparrow/arrow_interface/arrow_array_schema_info_utils.hpp

```diff
@@ -19,15 +19,7 @@
 
 namespace sparrow
 {
-
-    /// @returns `true` if  the format of an `ArrowArray` for a given data type is valid, `false` otherwise.
-    [[nodiscard]] inline bool validate_format_with_arrow_array(data_type, const ArrowArray&)
-    {
-        return true;
-        /* THE CODE USED TO MAKES WRONG ASSUMPTIONS AND NEEDS TO BE REFACTORED IN A SEPERATE PR*/
-    }
```

Comment:
Should we open an issue to track this? Or has it become irrelevant?

Reply:
It was not used anymore, just remove unused code.